themes/default/content/templates/container-service/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Container Templates
+layout: overview
+description: Pulumi program templates are the fastest way to deploy [containerized applications](/containers) on [AWS](/aws), [Azure](/azure), or [Google Cloud Platform](/gcp). Templates come with predefined [infrastructure as code](/what-is/what-is-infrastructure-as-code) so you can get started instantly.
+meta_desc: Pulumi program templates that make it easy to deploy containerized applications on AWS, Azure, or Google Cloud Platform.
+meta_image: meta.png
+weight: 1
+---
+
+### What are containerized applications?
+
+Containerized applications run in isolated runtime environments called containers, which enable developers to package an application and its dependencies into a lightweight environment. Docker is most commonly used to create container images. Containerized applications are great for scenarios such as microservices, batch jobs, and migrating a legacy application. The benefits of containers include better scalability, portability, and fault isolation.
```

Comment:
I think what you need to do here is talk about Running containers and then introduce that these clouds of container services that allow you to run said containers atop compute. And then you present the different options. 

```suggestion
### What are containers?

Containers are lightweight virtualized operating system environments where developers can package and run an application and its dependencies as isolated runtime environments. The benefits of containers include better scalability, portability, and fault isolation. Docker is most commonly used to create container images. Containers are great for scenarios such as microservices, batch jobs, and migrating a legacy application. In the cloud, containers are typically run using container services that manage the container orchestration, cluster management, and compute for you. 
```